README.md

```diff
@@ -17,13 +17,13 @@ Bedrock also provides automation for deploying Kubernetes clusters with Terrafor
 
 A Bedrock deployment follows three general steps at a high level:
 
-1. Define a [Fabrikate](https://github.com/Microsoft/fabrikate) definition for your deployment.
-2. [Deploy a CI/CD pipeline](./gitops) to build resource manifests from this deployment definition.
-3. [Create and deploy](./cluster) a Kubernetes environment with Flux.
+1. [Create and deploy azure-simple](./docs/azure-simple/README.md) a Kubernetes environment that uses Flux.
```

Comment:
Instead of linking directly to the walkthrough - what I would suggest is that we add this as the first step recommended in the ~/cluster/azure documentation item once they have the overview.